src/css/custom.css

```diff
@@ -210,51 +210,51 @@ h4,
 }
 
 .related-read-archetype-tag {
-  padding:3px 6px;
+  padding: 3px 6px;
```

Comment:
Is prettier modifying CSS? Strange.

Reply:
Yep, that's just what happened when I ran `npm run format`